### ISSUE TITLE
Criteria notes are not shown

### ISSUE THREAD

**Issue by MarkoTodorovicFFG** [w3c/wcag-em-report-tool#522]
There are notes defined in WCAG (e.g. here https://www.w3.org/TR/WCAG21/#keyboard) and in `wcag2-en.json`, but the notes aren't shown when criterion text is toggled in step 4.

There is a typo in `/app/views/directives/successCriterion.html` which prevents this:
```
<p ng-if="details.type === 'note'">
<em>{{ "AUDIT.NOTE" | translate }}:</em> {{ details.text}}
</p>
```
should be changed to:
```
<p ng-if="detail.type === 'note'">
<em>{{ "AUDIT.NOTE" | translate }}:</em> {{ detail.text}}
</p>
```
("details" to "detail")
